app/tests/tests_10/test_external_assets_endpoint.py

```diff
@@ -0,0 +1,206 @@
+from parameterized import parameterized
+
+from django.test import Client
+
+from stac_api.models import Asset
+
+from tests.tests_10.base_test import StacBaseTestCase
+from tests.tests_10.data_factory import Factory
+from tests.tests_10.utils import reverse_version
+from tests.utils import client_login
+from tests.utils import mock_s3_asset_file
+
+
+class AssetsExternalAssetEndpointTestCase(StacBaseTestCase):
+
+    @mock_s3_asset_file
+    def setUp(self):  # pylint: disable=invalid-name
+        self.factory = Factory()
+        self.collection = self.factory.create_collection_sample().model
+        self.item = self.factory.create_item_sample(collection=self.collection).model
+        self.client = Client()
+        client_login(self.client)
+        self.maxDiff = None  # pylint: disable=invalid-name
+
+    def test_create_asset_with_external_url(self):
+        collection = self.collection
+        item = self.item
+
+        asset_data = {
+            'id': 'clouds.jpg',
+            'description': 'High in the sky',
+            'type': 'image/jpeg',  # specify the file explicitly
+            'href':
+                'https://sys-data.int.bgdi.ch/ch.meteoschweiz.ogd-swisstopo-test/ch.meteoschweiz.ogd-swisstopo-testitem/clouds.jpg'
```

Comment:
Exposed URL

Reply:
For this to work we rely on the presence of that file (which was uploaded manually) right? Could we upload that file during the setup? To really make sure it's there? (Unlikely that it will be deleted, but still...)

Reply:
I'm gonna try!! :+1: :)

Reply:
Since it's now available from the env, I don't see how to do this reliably from the code. It's now up to who or whatever triggers the test to make sure that the file is available.
Could maybe be done in the builder pipeline, for that matter.